src/app/watchlist/repositories/getWatchlistsRepositories.go

```diff
@@ -0,0 +1,37 @@
+package repositories
+
+import (
+	genericConstants "stock_broker_application/src/constants"
+	"stock_broker_application/src/models"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+type GetWatchlistsRepository interface {
+	GetWatchlists(ctx *gin.Context) ([]string, error)
+}
+
+type userDBRepository struct {
+	DB *gorm.DB
+}
+
+func NewGetWatclistsRepository(db *gorm.DB) GetWatchlistsRepository {
+	return &userDBRepository{DB: db}
+}
+
+func (repository *userDBRepository) GetWatchlists(ctx *gin.Context) ([]string, error) {
+	var watchlistSlice []models.Watchlist
+	var watchlistNames []string
+	userID := ctx.Value(genericConstants.Id)
+
+	if err := repository.DB.Model(&models.TokenData{}).Where("user_id = ?", userID).Find(&watchlistSlice).Error; err != nil {
```

Comment:
again declare column "user_id" name in genericConstants
under already defined const group
// Column Name Constants
const (